### PR TITLE
Ability for polyserve to serve entry points other than index.html

### LINKED ISSUE #180: polyserve should serve the entry point configured in polymer.json
Currently it assumes index.html.

### PR DESCRIPTION
**Pull request by aomarks**
- Part of #180.
- Note that it won't be possible to use the feature unless you are calling via polymer-cli.

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by justinfagnani** [comment]
If this requires a change to polymer-build, doesn't it also need to update the polymer-build version constraint? Can you add a link to the PR or issues this depends on?

LGTM other than that.

File: src/start_server.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import * as assert from 'assert';
 import * as express from 'express';
 import * as fs from 'mz/fs';
 import * as path from 'path';
+import {urlFromPath} from 'polymer-build';
```
Comment (justinfagnani) [inline src/start_server.ts]: It is a little misplaced, but I don't really have a better idea at the moment. We don't have a library of utilities like this. Maybe just don't export out of the main polymer-build module, but import directly from `polymer-build/path-transformers`

Comment (aomarks) [inline src/start_server.ts]: SGTM Thanks!

**Comment by aomarks** [comment]
> If this requires a change to polymer-build, doesn't it also need to update the polymer-build version constraint? Can you add a link to the PR or issues this depends on?
> 
> LGTM other than that.

As discussed, importing directly from path-transformers now, so no dependency changes needed after all.

**Pull request merged**